client/src/components/video_detail/VideoDetail.tsx

```diff
@@ -73,16 +65,18 @@ const VideoDetail = (props: IProps) => {
   //                = t2 - t1 + t'1
 
   useEffect(() => {
-    const processTime: ProcessTime =
-      latestTimeGetVideo - playingVideo.latestUpdateAt + playingVideo.progress;
-    playerRef.current && playerRef.current.seekTo(processTime);
-  }, [playingVideo]);
-
-  const onReady = useCallback(() => {
-    socket.emit(RoomEvent.SELECT_VIDEO, { playingVideo, roomId });
-  }, [playingVideo.url]);
+    /** latestTimeGetVideo : time users get latest update of movie from server */
+    if (playingVideo) {
+      const latestTimeGetVideo = new Date().getTime() / 1000;
+      const processTime: ProcessTime =
+        latestTimeGetVideo -
+        playingVideo.latestUpdateAt +
+        playingVideo.progress;
+      playerRef.current && playerRef.current.seekTo(processTime, "seconds");
+    }
+  }, [playingVideo, playerRef]);
 
-  if (!playingVideo.url) {
+  if (!playingVideo?.url) {
```

Comment:
playingVideo is not nullable, can we remove the `?`

Reply:
fixed